docs/_static/credential/v1.yaml

```diff
@@ -0,0 +1,246 @@
+      operationId: "create"
+      tags:
+        - credential
+      responses:
+        "200":
+          description: "New DID has been created successfully. Returns the DID Document."
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/VerifiableCredential'
+        "400":
+          description: One or more of the given parameters are invalid
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+  /internal/credential/v1/vc/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: URL encoded ID.
+        required: true
+        example:
+          - "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467"
+        schema:
+          type: string
+    get:
+      summary: "Resolves a verifiable credential"
+      operationId: "resolve"
+      tags:
+        - credential
+      responses:
+        "200":
+          description: Credential has been found and is returned.
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/VerifiableCredential'
+        "404":
+          description: Corresponding credential could not be found.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+  /internal/credential/v1/revoke/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: URL encoded ID.
+        required: true
+        example:
+          - "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467"
+        schema:
+          type: string
+    post:
+      summary: "Revoke a credential"
+      operationId: "revoke"
+      tags:
+        - credential
+      responses:
+        "202":
+          description: Revocation has been accepted.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VerificationOutcome"
+        "400":
+          description: Credential can't be revoked. Most likely due to a missing private key.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: Credential is not found.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+  /internal/credential/v1/{concept}:
```

Comment:
Would it be viable to URL-encode it (`v1/{concept}?q={query}`)? Because then we can just make it a GET query, as expected for read operations for REST-y services

Reply:
It's subject to change when the API is implemented but the problem is that each concept has different query params. The idea is to accept the concept JSON as search body and create a query on given concept paths:

```
{
  "issuer": "X",
  "company": {
    "name": "query"
  }
}
```